crawl-ref/source/dat/des/arrival/small.des

```diff
@@ -3412,3 +3412,126 @@ MAP
   ...........
    .........
 ENDMAP
+
+##############################################################
+NAME:    heliophobic_arrival_battle_scene
+TAGS:    arrival no_monster_gen no_trap_gen no_pool_fixup
+ORIENT:  float
+ITEM:    human skeleton / dwarf skeleton , \
+         goblin skeleton / gnoll skeleton / \
+         kobold skeleton / orc skeleton
+SHUFFLE: cx
+SHUFFLE: tvw
```

Comment:
A common idiom to increase randomization is to make the whole set of floor you'd like to randomize some non . glyph, like `-`, then use `NSUBST:` to do substitutions. For example,

```
NSUBST: - = 6:d / 11:e / *:.
```

This will randomize your corpse placement.

Reply:
Actually, characters who are flying CAN be teleported over water:

https://github.com/crawl/crawl/issues/1234

And I am not thrilled with someone having to fly half way around my fort to exit the water at the `@` entrance end.  It just seem tedious and unnecessary, until such a time as flying players can't be teleported over water/lava, anyway.

Reply:
Great idea! Implemented!